src/components/victory-gauge.js

```diff
@@ -302,75 +321,176 @@ export default class VictoryGauge extends React.Component {
       .sort(null)
       .startAngle(degreesToRadians(props.startAngle))
       .endAngle(degreesToRadians(props.endAngle))
-      .padAngle(degreesToRadians(props.padAngle))
-      .value((datum) => { return datum.y; });
+      .padAngle(degreesToRadians(props.padAngle));
+  }
+  getSegments(props) {
+    const {segments} = props;
+    if (segments && segments.length) {
+      return segments;
+    }
+    // if there are no segments provide set of data as a
+    // segment that spans entire arc. array of 1?
+    return [1];
+  }
+  getGaugeRange(props, segmentLocations) {
+    const radiansToDegrees = (r) => r * (180 / Math.PI);
+    const {domain} = props;
+    return {
+      minimum: {
+        value: domain && domain[0] || segmentLocations[0].data,
+        degrees: radiansToDegrees(segmentLocations[0].startAngle)
+      },
+      maximum: {
+        value: domain && domain[1] || segmentLocations[1].data,
+        degrees: radiansToDegrees(segmentLocations.reverse()[0].endAngle)
```

Comment:
I agree. These issues are related to `tickValues`, `domain` and `segments`. The current implementation takes the values and sizes them according to their number (here we see 1 and 2 are half of the chart, and 3 represents the other half), and the total span of the chart represents the sum of those values (6 in this case). 

But I'm thinking `segments` should represent the endpoints of the segments; where the ticks are placed, making the max value 3, with segment breaks at 2 and 1, which makes entirely more sense but raises a couple questions:

Will first and last values of `segments` become the domain if domain isn't provided? If that's the case, would the above example become a 2 segment chart? I'm thinking it should.

If `domain` is provided along with `segments` is it used or should it only be used as a fallback in case `segments` isn't provided? If it will be used, and `segments` values are within the `domain` values, will ticks be placed at the end of the chart (as it does here), or only at the defined breaks in the `segments`?

Also, perhaps the best solution would be to use a similar approach to the `VictoryPie` API. If we did something like: `segments={[{x: 1, y: 'labelName'}, {x: 2, y: 20}, {x: 3, y: 30}]}` and did away with `tickValues` that would make for a cleaner and less confusing API that allows for direct mapping of the segment endpoint or no label at all.